spec/mocha.opts

```diff
@@ -0,0 +1,2 @@
+--require spec/setup.js
+--recursive
```

Comment:
http://mochajs.org/#mocha.opts

It just requires the setup file which puts assert and things on global and tells mocha to search sub directories for tests.